vuejs/index.html

```diff
@@ -0,0 +1,18 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <link rel="icon" type="image/svg+xml" href="/vite.svg" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>mathquill4quill demo (Vue 3)</title>
+  </head>
+  <body>
+    <div id="app"></div>
+    <script
+      src="https://code.jquery.com/jquery-3.7.1.min.js"
```

Comment:
Could you include the jquery dependency via an import from an npm module instead of a global include? I believe this would be more idiomatic for Vue. You can check out the React demo for an example ([here](https://github.com/c-w/mathquill4quill/blob/f96f4ab34acc9cb3b2c7e8e90301e8c3bfc320df/reactjs/src/jquery.js)).

Reply:
done